fmriprep/workflows/bold/outputs.py

```diff
@@ -82,7 +82,7 @@ def init_func_derivatives_wf(
                                    ('confounds_metadata', 'meta_dict')]),
     ])
 
-    if spaces.unique('output').intersection(('func', 'run', 'bold', 'boldref', 'sbref')):
+    if set(spaces.get_nonstd_spaces()).intersection(('func', 'run', 'bold', 'boldref', 'sbref')):
```

Comment:
computing `spaces.get_X_spaces()` multiple times feels unnecessary - we should just assign these values (std, nonstd, fs) to a variable